README.md

```diff
@@ -256,6 +256,6 @@ The following steps describe the minimal steps required to successfully log in t
 
 1. Run `uid2-admin` locally by following the README: https://github.com/IABTechLab/uid2-admin
 1. Ensure that the site ids of your participants exist in admin. That goes for the current participant you are logged in to, as well as the participants you are interacting (e.g. sharing) with. You can check the existing ids by looking at `sites.json` in `uid2-admin` or by going to http://localhost:8089/adm/site.html and hitting `List Sites`, given the service is running locally.
-1. Ensure that you have a key in `admins.json` in `uid2-admin` with the `SHARING_PORTAL` role, and that key is in your `.env` as `SSP_ADMIN_SERVICE_CLIENT_KEY`.
-   - You should be able to use an existing key in `admins.json`, rather than having to create a new one.
+1. Ensure that you have set the Okta client secret in your `.env` as `SSP_OKTA_CLIENT_SECRET`.
+   - You can find the key for local testing in 1Password under "Okta localhost deployment.
```

Comment:
How would this work for an open source contributor? I don't think we want to tie local dev to our main Okta instance.

Reply:
They can set is_auth_disabled = true in their admin config. Updated the readme to note this